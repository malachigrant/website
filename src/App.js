/** @jsx jsx */
import { jsx, css } from '@emotion/core';

export const App = () => {
  const Style = css`
    margin: 1em auto 0 auto;
    padding: 1em 2em;
    border-radius: 0.2em;
    &:hover {
      background-color: #000000;
      color: #ffffff;
    }
  `;
  return <div css={Style}>Welcome to my template!</div>;
};

export default App;
