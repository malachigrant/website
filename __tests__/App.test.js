/** @jsx jsx */
import { create } from 'react-test-renderer';
import { jsx } from '@emotion/core';

import App from 'App';

describe('App', () => {
  it('renders correctly', () => {
    const tree = create(<App />);
    expect(tree.toJSON()).toMatchSnapshot();
  });
});
