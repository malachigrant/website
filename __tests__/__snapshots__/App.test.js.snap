// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
<div
  className="css-1podhy4-Style"
>
  Welcome to my template!
</div>
`;
